events/discordEvents/client/ready.js

```diff
@@ -834,6 +834,52 @@ module.exports = {
 
             sendToChannelFeeds('postUpdate', data, client);
          });
+
+         // =============================================================
+         // Lil Nouns
+         // =============================================================
+         
+         Nouns.on('ProposalCreated', async data => {
+            data.description = data.description.substring(0, 500);
+
+            try {
+               const proposal = await Proposal.tryCreateProposal(data);
```

Comment:
`Proposal` is used to store information on Nouns DAO proposals. To avoid mixing stuff up, we could consider adding a new `LilNounsProposal` database collection and schema.

```suggestion
               const proposal = await LilNounsProposal.tryCreateProposal(data);
```